pages/home.vue

```diff
@@ -111,12 +111,14 @@ export default {
           name:          'name',
           labelKey:      'tableHeaders.name',
           sort:          ['nameSort'],
-          canBeVariable: true
+          canBeVariable: true,
+          search:        ['nameSort'],
```

Comment:
This cell would currently display nothing except because you're later providing a template for this cell.  All columns should have a `value` set, to `'nameDisplay'` in this case.  That would then also be used as the searchable field by default so you don't need to specify a `search`.

`nameSort` modifies the name so that it sorts more desirably (e.g. `cluster1 cluster2 ... cluster10` instead of `cluster1 cluster10 cluster2`) but you want to search the value that is actually displayed to the user and not what it's been transformed into to sort properly.